app/theme/server/server.js

```diff
@@ -145,45 +155,18 @@ export const theme = new class {
 	getCss() {
 		return settings.get('css') || '';
 	}
-
 };
 
-
 WebApp.rawConnectHandlers.use(function(req, res, next) {
 	const path = req.url.split('?')[0];
 	const prefix = __meteor_runtime_config__.ROOT_URL_PATH_PREFIX || '';
-	if (path === `${ prefix }/theme.css`) {
-		const css = theme.getCss();
-		const hash = crypto.createHash('sha1').update(css).digest('hex');
-		res.setHeader('Content-Type', 'text/css; charset=UTF-8');
-		res.setHeader('ETag', `"${ hash }"`);
-		res.write(css);
-		return res.end();
-	} else {
+	if (path !== `${ prefix }/theme.css`) {
 		return next();
 	}
-});
-
-const { calculateClientHash } = WebAppHashing;
 
-WebAppHashing.calculateClientHash = function(manifest, includeFilter, runtimeConfigOverride) {
-	const css = theme.getCss();
-	if (css.trim() !== '') {
-		const hash = crypto.createHash('sha1').update(css).digest('hex');
-		let themeManifestItem = _.find(manifest, function(item) {
-			return item.path === 'app/theme.css';
-		});
-		if (themeManifestItem == null) {
-			themeManifestItem = {};
-			manifest.push(themeManifestItem);
-		}
-		themeManifestItem.path = 'app/theme.css';
-		themeManifestItem.type = 'css';
-		themeManifestItem.cacheable = true;
-		themeManifestItem.where = 'client';
-		themeManifestItem.url = `/theme.css?${ hash }`;
-		themeManifestItem.size = css.length;
-		themeManifestItem.hash = hash;
-	}
-	return calculateClientHash.call(this, manifest, includeFilter, runtimeConfigOverride);
-};
+	res.setHeader('Content-Type', 'text/css; charset=UTF-8');
+	res.setHeader('Content-Length', currentSize);
+	res.setHeader('ETag', `"${ currentHash }"`);
+	res.write(theme.getCss());
```

Comment:
Would this actually exist?  

I know let is defined outside this scope... but will this actually persist?

Reply:
yes =)

> Variables declared by let have their scope in the block for which they are defined, as well as in any contained sub-blocks. 

[source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#Scoping_rules_2)

Reply:
actually, if you're worried about `theme` variable, it is a const that was [being used before](https://github.com/RocketChat/Rocket.Chat/pull/13953/files#diff-8041caddb7bb0c9a1cc6a67a9501d845L170) as well